shell.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import(
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"regexp"
+)
+
+// ShellCommand is a helper function that executes the specified shell command with its
+// specified arguments. It prints the output of the shell command.
+func ShellCommand(cmd string, args ...string) (out string, err error) {
+	var output []byte
+
+	output, err = exec.Command(cmd, args...).CombinedOutput()
+	if err != nil {
+		return "", err
+	}
+
+	return string(output), err
```

Comment:
Seems like people could just call the exec.Command version and skip the middleman.